test/test-applications/integrations/TestApplication.NLogBridge/NLogLogger.cs

```diff
@@ -0,0 +1,51 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+using Microsoft.Extensions.Logging;
+using NLog;
+
+namespace TestApplication.NLogBridge;
+
+internal class NLogLogger : Microsoft.Extensions.Logging.ILogger
```

Comment:
I would consider using this package https://github.com/NLog/NLog.Extensions.Logging
It is common ILogger provider.
log4net implements it in our repository, because similar package was not available at the moment of writing.